app/views/shared/_header.html.erb

```diff
@@ -19,7 +19,13 @@
         </ul>
         <ul class="nav navbar-nav ml-auto">
             <li class="nav-item">
-                <a class="nav-link" href="#">Logowanie/Rejestracja</a>
+
+                  <% if current_user %> 
+                    <%= current_user[:display_name] %>
+                    <%= link_to "Logout", destroy_user_session_path, method: :delete %>
```

Comment:
are you sure this should be `:delete` method? What if we want to delete user?

Reply:
In this case :delete is used for delete current user session. If you whant delete user you have to use different path "user_registration_path".

Reply:
ok thanks